urnik/templates/izbira_semestra.html

```diff
@@ -9,9 +9,9 @@
     {% for semester in semestri %}
     <li>
       {% if forloop.counter == semestri|length %}
```

Comment:
Tole je točno `forloop.last` (vem, da ni tvoje, ampak vseeno sem ravno opazil)

Reply:
Take malenkosti se da baje popravit kar direktno iz komentarja na PR. Pa poskusimo:
```suggestion
      {% if forloop.last %}
```

Reply:
Jap, deluje - samo en presledek je očitno pogoltnilo :/